MoviesStoreApp/MVPMovies/NetworkService/KeyChainService.swift

```diff
@@ -0,0 +1,31 @@
+// KeyChainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import KeychainSwift
+
+/// Сервис сохранения APIKey
+final class KeyChainService {
+    // MARK: - Public Properties
+
+    static let shared = KeyChainService()
+
+    // MARK: - Private Init
+
+    private init() {}
+
+    // MARK: - Public Methods
+
+    func getAPIKey() -> String {
+        guard let apiKey = KeychainSwift().get(ConstantsKeyChain.key) else {
+            saveAPIKey()
+            return getAPIKey()
+        }
+        return apiKey
+    }
+
+    // MARK: - Private Methods
+
+    private func saveAPIKey() {
+        KeychainSwift().set(URLRequest.apiKey, forKey: ConstantsKeyChain.key)
```

Comment:
```suggestion
        KeychainSwift().set(, forKey: ConstantsKeyChain.key)
```
мы не храним в явном виде данные в приложении